src/ManageCourses.Api/ManageCourses.Api.csproj

```diff
@@ -28,13 +28,18 @@
     <ProjectReference Include="..\ManageCourses.Domain\ManageCourses.Domain.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Content Update="appsettings.json">
+    <None Update="app_offline.htm.example">
```

Comment:
Why?

Reply:
moved from below because csproj is unstable, meh